api-client/src/camera/getCameraSettings.ts

```diff
@@ -0,0 +1,19 @@
+import { GET, request } from '../request'
+
+import type { CameraId } from '@opentrons/shared-data'
+import type { CameraResponse, CameraSettings } from '../camera'
+import type { ResponsePromise } from '../request'
+import type { HostConfig } from '../types'
+
+export function getCameraSettings(
+  config: HostConfig,
+  data: CameraSettings,
+  cameraId: CameraId
+): ResponsePromise<CameraResponse> {
```

Comment:
```suggestion
): ResponsePromise<CameraSettingsResponse> {
```
See comment elsewhere